Dockerfile

```diff
@@ -21,6 +21,11 @@ RUN set -x \
       | egrep -o '/screwdriver-cd/log-service/releases/download/v[0-9.]*/logservice' \
       | wget --base=http://github.com/ -i - -O logservice \
    && chmod +x logservice \
+   # Download Meta CLI
+   && wget -q -O - https://github.com/screwdriver-cd/meta-cli/releases/latest \
+      | egrep -o '/screwdriver-cd/meta-cli/releases/download/v[0-9.]*/meta-cli' \
+      | wget --base=http://github.com/ -i - -O meta-cli\
```

Comment:
@tk3fftk Hi Hiroki, when u release the `meta-cli`, you build the binary with the name `meta`
 https://github.com/screwdriver-cd/meta-cli/releases